ha_cluster_exporter.go

```diff
@@ -99,6 +99,10 @@ func main() {
 		log.Info("DRBD collector registered")
 	}
 
+	if !log.IsLevelEnabled(log.DebugLevel) {
```

Comment:
can we add here a comment explaining what we do?

Reply:
done!